Build failure: wsmancli / openwsman / sblim-sfcc

## Steps To Reproduce

1. Install wsmancli via whichever mechanism.

## Build log

```
error: builder for '/nix/store/40cakx19c8zaznrv772pyff2lasnmyya-sblim-sfcc-2.2.9.drv' failed with exit code 2;
       last 25 log lines:
       > backend/cimxml/cimXmlParser.c: In function 'scanCimXmlResponse':
       > backend/cimxml/cimXmlParser.c:1352:25: error: implicit declaration of function 'startParsing' [8;;https://gcc.gnu.org/onlinedocs/gcc-14.2.0/gcc/Warning-Options.html#index-Wimplicit-function-declaration-Wimplicit-function-declaration8;;]
       >  1352 |    control.respHdr.rc = startParsing(&control);
       >       |                         ^~~~~~~~~~~~
       > backend/cimxml/cimXmlParser.c: At top level:
       > backend/cimxml/cimXmlParser.c:79:15: warning: 'c' is static but declared in inline function 'skipWS' which is not static
       >    79 |    static int c = 0;
       >       |               ^
       > backend/cimxml/client.c: In function 'genRequest':
       > backend/cimxml/client.c:408:4: warning: 'CURLOPT_PROGRESSFUNCTION' is deprecated: since 7.32.0. Use CURLOPT_XFERINFOFUNCTION [8;;https://gcc.gnu.org/onlinedocs/gcc-14.2.0/gcc/Warning-Options.html#index-Wdeprecated-declarations-Wdeprecated-declarations8;;]
       >   408 |    curl_easy_setopt(con->mHandle, CURLOPT_PROGRESSFUNCTION, checkProgress);
       >       |    ^~~~~~~~~~~~~~~~
       > In file included from backend/cimxml/client.c:26:
       > /nix/store/g26bs63hz87c9s3sg0v42d6d8gjk36qw-curl-8.11.0-dev/include/curl/curl.h:1303:3: note: declared here
       >  1303 |   CURLOPTDEPRECATED(CURLOPT_PROGRESSFUNCTION, CURLOPTTYPE_FUNCTIONPOINT, 56,
       >       |   ^~~~~~~~~~~~~~~~~
       > make[2]: *** [Makefile:875: libcimcClientXML_la-cimXmlParser.lo] Error 1
       > libtool: link: gcc -shared  -fPIC -DPIC  .libs/libcimcclient_la-cimcclient.o   -ldl -lpthread  -g -O2 -Wl,--version-script -Wl,./cimc/libcimcclient.Versions   -Wl,-soname -Wl,libcimcclient.so.0 -o .libs/libcimcclient.so.0.0.0
       > libtool: link: (cd ".libs" && rm -f "libcimcclient.so.0" && ln -s "libcimcclient.so.0.0.0" "libcimcclient.so.0")
       > libtool: link: (cd ".libs" && rm -f "libcimcclient.so" && ln -s "libcimcclient.so.0.0.0" "libcimcclient.so")
       > libtool: link: ( cd ".libs" && rm -f "libcimcclient.la" && ln -s "../libcimcclient.la" "libcimcclient.la" )
       > make[2]: Leaving directory '/build/source'
       > make[1]: *** [Makefile:1041: all-recursive] Error 1
       > make[1]: Leaving directory '/build/source'
       > make: *** [Makefile:586: all] Error 2
       For full logs, run 'nix log /nix/store/40cakx19c8zaznrv772pyff2lasnmyya-sblim-sfcc-2.2.9.drv'.
error: 1 dependencies of derivation '/nix/store/848nv01jbf4m2l09hyvq46rjad2d8mqg-openwsman-2.7.2.drv' failed to build
error: 1 dependencies of derivation '/nix/store/6iis2rm9731cv305h7pfnnrrvv6a7295-wsmancli-2.6.2.drv' failed to build
```

Full build failure log at https://gist.github.com/nipsy/82827f1bdae491e45b17fe50452a58d6.

## Metadata

```
# nix-shell -p nix-info --run "nix-info -m"
 - system: `"x86_64-linux"`
 - host os: `Linux 6.12.5, NixOS, 25.05 (Warbler), 25.05.20241219.d70bd19`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.24.11`
 - channels(root): `"nixos"`
 - nixpkgs: `/nix/store/8fwsiv0hd7nw1brkvka0jf1frk3m7qkr-source`
```

## Notify maintainers

@deepfire 

---

Note for maintainers: Please tag this issue in your PR.

---

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

--- a/pkgs/by-name/sb/sblim-sfcc/package.nix
+++ b/pkgs/by-name/sb/sblim-sfcc/package.nix
@@ -8,13 +8,13 @@
 
 stdenv.mkDerivation rec {
   pname = "sblim-sfcc";
-  version = "2.2.9"; # this is technically 2.2.9-preview
+  version = "2.2.8-unstable-2023-06-26";
 
   src = fetchFromGitHub {
     owner = "kkaempf";
     repo = "sblim-sfcc";
-    rev = "514a76af2020fd6dc6fc380df76cbe27786a76a2";
-    sha256 = "06c1mskl9ixbf26v88w0lvn6v2xd6n5f0jd5mckqrn9j4vmh70hs";
+    rev = "881fccbaf19e26cbef3da1bebe2b42b3a9de1147";
+    hash = "sha256-zXQD+IYuMV5vw27FpTpeCfh/mf0wvKzOvc4bplEDJCw=";
   };
 
   buildInputs = [ curl ];

